python/idsse_common/idsse/common/utils.py

```diff
@@ -225,6 +232,48 @@ def round_half_away(number: Scalar, precision: int = 0) -> Scalar:
     return int(rounded_number) if precision == 0 else float(rounded_number)
 
 
+def round_(
+    number: int | float,
+    precision: int = 0,
+    rounding: RoundingParam = RoundingMethod.ROUND
+) -> int | float:
+    """
+    Round a float to a set number of decimal places, using "ties away from zero" method if rounding
+    is RoundingMethod.ROUND or math.floor() if RoundingMethod.FLOOR.
+
+    This behavior contrasts with Python 3's built-in round() or numpy.round() functions,
+    which use "ties to even" method.
+
+    | Input | round() | round_half_away() |
+    | ----- | ------- | ----------------- |
+    |   2.5 |       2 |                 3 |
+    | -14.5 |     -14 |               -15 |
+
+    Args:
+        number (int | float | numpy.int | numpy.float_): a number, often from a Python/numpy array
```

Comment:
This type hinting is imperfect--numpy types like int, float32, etc. work too--but we didn't want to import numpy into this module just for the type hints.